pages/services/data-science-engine/1.0.0/examples/index.md

```diff
@@ -28,36 +129,63 @@ for (r in 1:nrow(mat)) {
-As mentioned in the Kerberos section, `{{ model.packageName }}` requires a {{ model.packageName }} file, the `krb5.conf`, and the keytab.
-An example of a {{ model.packageName }} file is:
-
-    KafkaClient {
-        com.sun.security.auth.module.Krb5LoginModule required
-        useKeyTab=true
-        storeKey=true
-        keyTab="/mnt/mesos/sandbox/kafka-client.keytab"
-        useTicketCache=false
-        serviceName="kafka"
-        principal="client@LOCAL";
-    };
-
-The corresponding `dcos {{ model.packageName }}` command would be:
-
-    dcos {{ model.packageName }} run --submit-args="\
-    --conf {{ model.packageName }}.mesos.containerizer=mesos \  # required for secrets
-    --conf {{ model.packageName }}.mesos.uris=<URI_of_{{ model.packageName }}.conf> \
-    --conf {{ model.packageName }}.mesos.driver.secret.names={{ model.packageName }}/__dcos_base64___keytab \  # base64 encoding of binary secrets required in DC/OS 1.10 or lower
-    --conf {{ model.packageName }}.mesos.driver.secret.filenames=kafka-client.keytab \
-    --conf {{ model.packageName }}.mesos.executor.secret.names={{ model.packageName }}/__dcos_base64___keytab \
-    --conf {{ model.packageName }}.mesos.executor.secret.filenames=kafka-client.keytab \
-    --conf {{ model.packageName }}.mesos.task.labels=DCOS_SPACE:/{{ model.packageName }} \
-    --conf {{ model.packageName }}.scheduler.minRegisteredResourcesRatio=1.0 \
-    --conf {{ model.packageName }}.executorEnv.KRB5_CONFIG_BASE64=W2xpYmRlZmF1bHRzXQpkZWZhdWx0X3JlYWxtID0gTE9DQUwKCltyZWFsbXNdCiAgTE9DQUwgPSB7CiAgICBrZGMgPSBrZGMubWFyYXRob24uYXV0b2lwLmRjb3MudGhpc2Rjb3MuZGlyZWN0b3J5OjI1MDAKICB9Cg== \
-    --conf {{ model.packageName }}.mesos.driverEnv.KRB5_CONFIG_BASE64=W2xpYmRlZmF1bHRzXQpkZWZhdWx0X3JlYWxtID0gTE9DQUwKCltyZWFsbXNdCiAgTE9DQUwgPSB7CiAgICBrZGMgPSBrZGMubWFyYXRob24uYXV0b2lwLmRjb3MudGhpc2Rjb3MuZGlyZWN0b3J5OjI1MDAKICB9Cg== \
-    --class MyAppClass <URL_of_jar> [application args]"
+```groovy
+def sum = 0
+1.upto(100) {
+    sum = sum + it
+}
+sum
+```
 
+## Example of Kotlin Kernel
+Open a `Kotlin Notebook` and put the following sections in a different code cells.
 
+```kotlin
+fun initMatrix(nrow: Int, ncol: Int): Array<IntArray> {
+    val mat = Array(nrow, {IntArray(ncol)})
+    var counter = 1
+    for(i in 0..9) {
+        for(j in 0..9) {
+            mat[i][j] = counter
+            counter += 1
+        }
+    }
+    return mat
+}
+```
+```kotlin
+fun sumMatrix(mat: Array<IntArray>): Int {
+    var sum = 0
+    for(i in 0..9) {
+        for(j in 0..9) {
+            sum += mat[i][j]
+        }
+    }
+    return sum
+}
+```
+```kotlin
+val mat = initMatrix(10, 10)
+val sum = sumMatrix(mat)
+sum
+```
+
+# Advanced
+
+## Example of launching Spark Job
+Open a `Python Notebook` and put the following code in a code cell.
+
+```bash
+! spark-submit --class org.apache.spark.examples.SparkPi http://downloads.mesosphere.com/spark/assets/spark-examples_2.11-2.4.0.jar 100
+```
```

Comment:
Can we replace this example with the one from Pandas, where we create Spark context programmatically Located in pandas_udf_with_arrow.ipynb

Calling a shell from a notebook cell is a workaround we use for tests, Terminal kernel should be used for this kind of calls. Let's add a separate Terminal kernel example instead.